### ISSUE TITLE
HA 2025.1b0 Failed to setup Rivian integration due to cryptography version

### ISSUE THREAD

**Issue by linuxkidd** [bretterer/rivian-python-client#117]
Just upgraded Home-Assistant to 2025.1b0 and hit the following cryptography version issue:
```
Unable to install package rivian-python-client[ble]==1.3.0: × No solution found when resolving dependencies:
 ╰─▶ Because rivian-python-client==1.3.0 depends on cryptography>=41.0.1,<44.0 and cryptography==44.0.0, 
we can conclude that rivian-python-client==1.3.0 cannot be used. 
And because you require rivian-python-client[ble]==1.3.0, we can conclude that your requirements are unsatisfiable.
```

Thanks! :)

**Comment by natekspencer** [comment]
This is resolved in [1.3.4](https://github.com/bretterer/home-assistant-rivian/releases/tag/1.3.4)

**Comment by linuxkidd** [comment]
Can confirm, issue is resolved. Closing the issue.

Thanks!
